server/container_create.go

```diff
@@ -607,6 +611,11 @@ func (s *Server) CreateContainer(ctx context.Context, req *pb.CreateContainerReq
 
 	newContainer.SetCreated()
 
+	if ctx.Err() == context.Canceled || ctx.Err() == context.DeadlineExceeded {
```

Comment:
We could move the context check to the top of the function into a deferred and catch the context cancelled or deadline exceeded anywhere. it would be beneficial to debug.PrintStack() within that function as well to print the stack. It would give us function that the context cancelled or exceeded on.

Reply:
Fixed